inference-engine/src/vpu/common/src/ngraph/transformations/merge_gather_gather_elements.cpp

```diff
@@ -0,0 +1,156 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "vpu/ngraph/transformations/merge_gather_gather_elements.hpp"
+
+#include <vpu/ngraph/operations/exp_gather_elements.hpp>
+#include <vpu/ngraph/utilities.hpp>
+
+#include <ngraph/pattern/op/wrap_type.hpp>
+#include <ngraph/opsets/opset6.hpp>
+#include <ngraph/pattern/matcher.hpp>
+#include <ngraph/pattern/op/label.hpp>
+#include <ngraph/validation_util.hpp>
+#include <ngraph/log.hpp>
+
+#include <numeric>
+
+NGRAPH_RTTI_DEFINITION(vpu::MergeGatherGatherElements, "MergeGatherGatherElements", 0);
+
+namespace vpu {
+
+bool MergeGatherGatherElements::run_on_function(std::shared_ptr<ngraph::Function> f) {
+    bool wasGraphChanged = false;
+
+    const auto gatherData = ngraph::pattern::any_input();
+    const auto gatherIndices = ngraph::pattern::any_input();
+    const auto gather = ngraph::pattern::wrap_type<ngraph::opset6::Gather>(
+        {gatherData, gatherIndices, ngraph::pattern::any_input()},
+        ngraph::pattern::consumers_count(1));
+
+    const auto squeezeAxes = ngraph::pattern::any_input(
```

Comment:
Just use wrap_type with Constant type.
```suggestion
    const auto squeezeAxes = ngraph::pattern::wrap_type<opset6::Constant>();
```

Reply:
Done.